launcher/settingsView/csettingsview_moc.cpp

```diff
@@ -515,23 +569,22 @@ void CSettingsView::on_comboBoxAlliedPlayerAI_currentTextChanged(const QString &
 	node->String() = arg1.toUtf8().data();
 }
 
-void CSettingsView::on_checkBoxAutoSavePrefix_stateChanged(int arg1)
+void CSettingsView::on_spinBoxAutosaveFrequency_valueChanged(int arg1)
 {
-    Settings node = settings.write["general"]["useSavePrefix"];
-    node->Bool() = arg1;
-    ui->lineEditAutoSavePrefix->setEnabled(arg1);
+    Settings node = settings.write["general"]["autosaveFrequency"];
+    node->Float() = arg1;
```

Comment:
Why float not integer?

Reply:
because other controls use the same. Copy paste -> changed to Integet